src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,8 +1,37 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
+    public static final DateTimeFormatter DATA_FORMAT =
+            DateTimeFormatter.ofPattern("dd.MM.yyyy");
+    public static final int INDEX_DATE = 0;
+    public static final int INDEX_NAME = 1;
+    public static final int INDEX_PRICE = 2;
+    public static final int INDEX_HOURS = 3;
 
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        final LocalDate dateFromLocal = LocalDate.parse(dateFrom, DATA_FORMAT);
+        final LocalDate dateToLocal = LocalDate.parse(dateTo, DATA_FORMAT);
+        LocalDate dateWork;
+        String[] dataOfMen = new String[4];
```

Comment:
do you need to set it to `new String[4]`?
let's just declare this variable later - when it is first used